### ISSUE TITLE
Flutter build error after ^2.2.0 upgrade

### ISSUE THREAD

**Issue by danialb007** [cafebazaar/flutter_poolakey#19]
This is the error output of gradle build.

```
Execution failed for task ':flutter_poolakey:generateReleaseRFile'.
> Could not resolve all files for configuration ':flutter_poolakey:releaseCompileClasspath'
```

**Comment by iranandroid** [comment]
@danialb007  
Hi, for me fix with add below line to build.gradle(Project) and sync project
`jcenter()
maven { url 'https://jitpack.io' }`

**Comment by nasrabadiAM** [comment]
Duplicate #24 and fixed in  [afc744a](https://github.com/cafebazaar/flutter_poolakey/commit/b76d6c95bd8a056ed9e2b632537a8528cafc744a).
